src/pages/CardNew.js

```diff
@@ -1,16 +1,50 @@
 import React, { useState } from 'react'
 import { Form, Input, Label, Button, FormGroup } from 'reactstrap'
-// import { useNavigate } from 'react-router-dom' || TODO vvvv
+import { useNavigate } from 'react-router-dom' 
 import './CardNew.css'
 
-const CardNew = () => {
-    return ( 
+const CardNew = ({createCard, deck}) => {
+
+    const navigate = useNavigate()
+    const [newCard, setNewCard] = useState({
+        question: '',
+        answer: '',
+        // deck_id: deck.id || TODO Uncomm
```

Comment:
Please either remove or implement it into your code.  My guess is that you will likely need the deck_id to create a new card.